docs/common/markdown.vue

```diff
@@ -49,6 +49,9 @@ watch(isMounted, value => {
 
   p {
     margin: 14px 0;
+    font-size: 17px;
```

Comment:
I mean should not add these styles, so you need to remove it, includes `font-size`, `line-height` and `letter-spacing`.

Reply:
ok